blog/2021-07-17-introducing-metriql.md

```diff
@@ -0,0 +1,94 @@
+---
+slug: introducing-metriql
+title: Introducing metriql
+author: Burak Kabakci
+author_title: Core Team
+author_url: https://github.com/buremba
+author_image_url: https://avatars.githubusercontent.com/u/82745
+tags: [announcement]
+hide_table_of_contents: true
+---
+
+Metric definitions is a hot topic most people are talking about lately. Ben has written [The missing piece of the modern data stack](https://benn.substack.com/p/metrics-layer) and Basecamp came up with the [Headless BI](https://basecase.vc/blog/headless-bi) concept in the last few months.
+
+We have been also giving careful thoughts to this particular problem for a while. The way we think about metrics is not just about the definitions, instead, a better integration between data tools and data warehouses. Most companies already use a modern data warehouse and have adopted the data warehouse as the single source of truth. They ingest data in real-time, run ELT jobs directly in their data warehouse, run ad-hoc queries and get the results in low-latency, and even run ML jobs on them.
+
+If you’re using a data transformation tool such as dbt, you can model your data and expose it to your data tools but that’s not often enough for the data tools, we often found ourselves doing extra data modeling in data tools for metric definitions. This is primarily because most of the data tools introduce different ways to define metrics and you need to adopt it to be able to analyze the data reliably. Let's categorize the metric types under 3 category:
```

Comment:
Let's categorize the metric types under 3 category: Burda bir aciklama lazim gibi geliyor, bir onceki paragrafta neden categorize etmeye gectik onu anlayamadim? metric type bir de doğrular mu?

Reply:
bu article icin bi tik teknik mi kaciyo bunlar yoksa oncesine soyle bisey ekleyebilirim:

In order to demonstrate the usual workflow of current BI tools, let's talk about three different metric types that we identified:

Reply:
```suggestion
If you’re using a data transformation tool such as dbt, you can model your data and expose it to your data tools but that’s not often enough for the data tools, we often found ourselves doing extra data modeling in data tools for metric definitions. This is primarily because most of the data tools introduce different ways and languages to define metrics and you need to adopt it to be able to analyze the data reliably using these tools. There are 3 main metric definition types that we've seen in most companies we've worked with, let me explain what they are and how they're defined differently in each BI tool:
```